src/components/SecondaryPanes/Frames.js

```diff
@@ -55,6 +55,14 @@ const Frames = createClass({
       || showAllFrames !== nextState.showAllFrames;
   },
 
+  componentWillMount() {
+    document.addEventListener("keyup", this.onKeyUp, false);
+  },
+
+  componentWillUnmount() {
+    document.removeEventListener("keyup", this.onKeyUp, false);
```

Comment:
Hmm, you should try to move this to `renderFrameList` where the UL is render and add attribute: `onKeyUp`